web/Web/web/static/js/document-viewer.js

```diff
@@ -703,7 +763,7 @@ docView.prototype = {
           'doc_search_snippet': "",
           'relevance': null,
           'additional_judging_criteria': additional_judging_criteria,
-          'source': options.judgingSourceName,
+          'source': parent.previouslyJudgedDocs[docid]["source"],
```

Comment:
The way I have designed the judgment table is that `source` will keep track of the source of the judgment made for a particular document. Right now, the source can be `CAL`, `search_SERP`,  `search`, `judgmentPage`. If we are making a new judgment for a document that has already been judged (see [here](https://github.com/UWaterlooIR/gathera/blob/master/web/Web/web/judgment/views.py#L70)), I update the source to have the latest source value, and I add append to `historyVerbose` the `historyItem` to keep track of changes. 

Instead of keeping the source the same as you are doing, just keep it `options.judgingSourceName` and in the `review.html` page, add `source:"reviewPage"` when setting the options for `var dv = new docView();`

Reply:
Done